android/src/main/java/pmstation/MainActivity.java

```diff
@@ -386,12 +408,18 @@ void setStatus(boolean connected) {
         menu.getItem(1).setVisible(!connected);
     }
 
+    @SuppressLint("UnspecifiedRegisterReceiverFlag")
```

Comment:
I don't like it, but I don't konw how to resolve it. The Android Linter throws:
`usbReceiver is missing RECEIVER_EXPORTED or RECEIVER_NOT_EXPORTED flag for unprotected broadcasts registered for pmstation.plantower`

Reply:
@sanchin can you advice?

Reply:
Suppressing will lead to runtime crashes on particular devices - different OS versions will behave differently